Description: force bundle to use bcrypt version 3.1.12 
 The build failed on armhf and armel if bundle uses bcrypt 3.1.13 
Author: Sophie Brun <sophie@offensive-security.com>
Last-Update: 2019-06-07
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/metasploit-framework.gemspec
+++ b/metasploit-framework.gemspec
@@ -53,7 +53,7 @@ Gem::Specification.new do |spec|
   # Backports Ruby features across language versions
   spec.add_runtime_dependency 'backports'
   # Needed for some admin modules (cfme_manageiq_evm_pass_reset.rb)
-  spec.add_runtime_dependency 'bcrypt'
+  spec.add_runtime_dependency 'bcrypt', '3.1.12'
   # Needed for Javascript obfuscation
   spec.add_runtime_dependency 'jsobfu'
   # Needed for some admin modules (scrutinizer_add_user.rb)
